cpp/tests/strings/array_tests.cu

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/column/column_factories.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/strings/sorting.hpp>
+#include <cudf/strings/copying.hpp>
+#include <cudf/types.hpp>
+
+#include <tests/utilities/base_fixture.hpp>
+#include "./utilities.h"
+
+#include <vector>
+#include <gmock/gmock.h>
+
+
+struct ArrayTest : public cudf::test::BaseFixture {};
+
+TEST_F(ArrayTest, Sort)
+{
+    std::vector<const char*> h_strings{ "eee", "bb", nullptr, "", "aa", "bbb", "ééé" };
+    std::vector<const char*> h_expected{ nullptr, "", "aa", "bb", "bbb", "eee", "ééé" };
+
+    auto d_strings = cudf::test::create_strings_column(h_strings);
+    auto strings_view = cudf::strings_column_view(d_strings->view());
+
+    auto results = cudf::strings::detail::sort(strings_view, cudf::strings::detail::name);
+    auto results_view = cudf::strings_column_view(results->view());
+
+    auto d_expected = cudf::test::create_strings_column(h_expected);
+    auto expected_view = cudf::strings_column_view(d_expected->view());
+
+    cudf::test::expect_strings_columns_equal(results_view, expected_view);
+}
+
+class ArrayTestParms1 : public ArrayTest,
```

Comment:
Use a descriptive name for what is being parameterized. Slice_Size? What does the `1` indicate?